containers/datalab/content/run.sh

```diff
@@ -206,20 +206,20 @@ mkdir -p /content/datalab/notebooks
 
 # Fetch docs and tutorials. This should not abort startup if it fails
 {
-if [ -d /content/datalab/docs ]; then
+if [ -d /content/datalab/ga ]; then
```

Comment:
Without changing the directory, we need to deal with the case where the docs dir is checked out to master (since almost all of our dev machine has ~/datalab/docs and is checked out to master), and we probably to fetch ga, checkout ga, and then rest local ga. It will be different from the release, where it simply pulls all notebook from master. Compared to adding an extra command, it seems doing it in a different dir is "closer" to the final release. No?

Also, if we want to keep master clean, we can create a new branch of datalab that pulls notebooks from ga branch. I am trying to avoid an extra branch though, but that means master from now on will pull notebooks from ga until notebooks are merged back to master.